test/test_grid.py

```diff
@@ -778,3 +778,85 @@ def test_build_face_edges_connectivity_fillvalues(self):
         self.assertTrue(
             np.array_equal(res_face_nodes_connectivity,
                            uds._ds["Mesh2_face_nodes"].values))
+
+    def test_node_face_connectivity_from_verts(self):
+        """Test generating Grid.Mesh2_node_faces from array input."""
+
+        # We used the following codes to generate the testing face_nodes_connectivity in lonlat,
+        # The index of the nodes here is just for generation purpose and ensure the topology.
+        # These information will not be used in the Grid.
+        # The input will be re-ordered inside the `Grid.build_face_nodes_connectivity` while the topology is the same.
+        # When checking with the expectation, we need to make sure the `Grid._node_face_connectivity` coheres
+        # with the `Grid._face_nodes_connectivity`.
```

Comment:
How about changing the whole above comment with the following:

"This nodes list is only for vertices creation purposes and the nodes' order will not be used the 
same in the `Grid` object; i.e. the `Grid` object instantiation will instead use the below 
`face_nodes_conn_lonlat ` connectivity variable and determine the actual node orders by itself."